.github/workflows/node.js.yml

```diff
@@ -1,6 +1,3 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```

Comment:
The link in this comment is dead, and the text is not really correct as there is no 'build' step, so I've chosen to remove this entirely. The names of the jobs and steps should be enough documentation for future reference.